source/Applications/Basic/FileFormats/ZDF2PLY/Properties/AssemblyInfo.cs

```diff
@@ -30,7 +31,7 @@
 //      Revision
 //
 // You can specify all the values or you can default the Build and Revision Numbers
-// by using the '*' as shown below:
-// [assembly: AssemblyVersion("1.0.*")]
```

Comment:
Not sure if you should remove these 2 comment lines. I would keep the comments as it was. Or then you need to remove the line above as well, now you have just removed half the sentence.

Reply:
Done